ckan/public/base/css/main.css

```diff
@@ -5560,6 +5560,9 @@ a.tag:hover {
 .js .tab-content.active {
   display: block;
 }
+.js .datetime {
```

Comment:
this class has an innocuous name that's also hard-to-grep-for, but it really should be shouting about how it does some spiffy JS magic and be easy to find in the code. Consider ".automatic-local-datetime" or similar